src/V3EmitMk.cpp

```diff
@@ -240,12 +240,10 @@ class EmitMk final {
         if (v3Global.opt.exe()) {
             of.puts("\n### Link rules... (from --exe)\n");
             of.puts(v3Global.opt.exeName()
-                    + ": $(VK_USER_OBJS) $(VK_GLOBAL_OBJS) $(VM_PREFIX)__ALL.a $(VM_HIER_LIBS)\n");
+                    + ": $(VK_USER_OBJS) $(VK_GLOBAL_OBJS) $(VK_OBJS) $(VM_HIER_LIBS)\n");
```

Comment:
Please keep the rule to make __ALL.a (though won't be the default) so we don't break old user's (and some of my outside-verilator) makefiles.  Make sure at least one test uses it.